app/src/main/java/org/mifos/selfserviceapp/data/Transaction.java

```diff
@@ -0,0 +1,70 @@
+package org.mifos.selfserviceapp.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Vishwajeet
+ * @since 10/8/16.
+ */
+public class Transaction {
+    private int id;
+    private List<Transaction> pageItems = new ArrayList<Transaction>();
```

Comment:
same refactoring needed as accounts.. move this list to something like TransactionListResponse class.